docs/2_2_common_mechanisms.adoc

```diff
@@ -860,7 +864,7 @@ include::../headers/fmi3FunctionTypes.h[tag=SetIntervalFraction]
 * `valueReferences` is an array of size `nValueReferences` holding the value references of the Clock variables.
 
 [[counters,`counters`]]
-* <<counters>> _[Note: This variable may increment by values other than 1.]_ and
```

Comment:
Why remove this note?

Reply:
This variable is never incremented. The implementation in the importer tool may increment some counter, but definitely not the "counter" argument of this function.
Furthermore there also may not be a counter in the importer tool implementation altogether. The FMI does not assume the implementation.

Reply:
Of course it can be increment it, but it can also be change with any increment (or decrement). Isn't that what the note is there to clarify?

Reply:
```C
typedef fmi3Status fmi3SetIntervalFractionTYPE(fmi3Instance instance,
                                               const fmi3ValueReference valueReferences[],
                                               size_t nValueReferences,
                                               const fmi3UInt64 counters[],
                                               const fmi3UInt64 resolutions[]);
```
It is const

Reply:
That doesn't prevent the importing tool from changing it between calls to `fmiSetInteralFractionType` (if the `intervalVariability` is `tunable`).

Reply:
Given that I think this note was added after people were confused on the rules,  I don't think removing it is a sensible way to go. We might want to adjust it going forward.